staff/lucas-wloch/challenges/calculadora-app/index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>EJ Calculadora</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="calculate.js" defer></script>
+    <script src="main.js" defer></script>
```

Comment:
script src=... should be at the end, just before body closes

Reply:
done